### ISSUE TITLE
Idea / improvement

### ISSUE THREAD

**Issue by ged-lendinvest** [QuatroCode/simplr-gulp#13]
index.html could have CSS file included:
<link rel="stylesheet" href="app/app.css" type="text/css" media="all"/>

**Comment by DovydasNavickas** [comment]
You mean bundling CSS into a single file and adding it into `index.html`?

**Comment by ged-lendinvest** [comment]
Yes, however if your approach is different, some guide would be awesome :)

**Comment by DovydasNavickas** [comment]
Well, in our case we lazily load all of the CSS files by importing them with SystemJS, which benefits us with faster initial load times. Bundling all of the CSS into one file is a bid of an "old story". Nowadays, with HTTP/2 already becoming available everywhere, there is no point of doing that.
